pom.xml

```diff
@@ -92,7 +92,7 @@
     <github.global.server>github</github.global.server>
     <google.auth.version>0.6.0</google.auth.version>
     <grpc.version>1.0.3</grpc.version>
-    <gax.version>0.6.0</gax.version>
+    <gax.version>0.6.1</gax.version>
```

Comment:
Now you don't even need this :-) Not that it isn't beneficial - let's keep the update anyway.